libnmstate/nm/profile.py

```diff
@@ -123,6 +123,7 @@ def _group_profile_by_action_order(self):
     def _prepare_state_for_profiles(self, net_state):
         _preapply_dns_fix_for_profiles(self._ctx, net_state)
         _mark_nm_external_subordinate_changed(self._ctx, net_state)
+        _mark_mode_changed_bond_child_interface_as_changed(net_state)
```

Comment:
What's wrong with codecov? It is strange that this line is not covered since there is a specific test that should use this code.

Reply:
Good question, it is on our TO-DO list to investigate what is going on with codecov. It is not working properly. I am going to create an issue